### ISSUE TITLE
Standard Field Namedimage Tile has low & fixed default size

### ISSUE THREAD

**Issue by jensens** [plone/plone.app.standardtiles#84]
see https://github.com/plone/plone.app.standardtiles/blob/master/plone/app/standardtiles/templates/namedimage.pt#L23

this is not very useful

**Comment by datakurre** [comment]
Agreed. Is that image field tile rendering? Custom image size would probably help some. Configuration for field tiles (for tile instance specific size) is completely unimplemented. “datakurre-custom” branch of Mosaic has support for custom field tiles with special registry configuration https://github.com/jyukopla/reclas/blob/master/resources/src/reclastheme/install/registry.xml#L69
